plugins/bookmarks/BookmarksSearchEngine.ts

```diff
@@ -49,21 +50,59 @@ export default class ChromiumBookmarksSearchEngine implements ISearchEngine {
 		}))
 	}
 
+	// TODO(davy): support macos
```

Comment:
thought: can we try `open` on macos? I think that should just work.

Reply:
yes, we can try this later